src/vs/editor/contrib/inlineCompletions/browser/ghostTextWidget.ts

```diff
@@ -157,6 +160,14 @@ export class GhostTextWidget extends Disposable {
 			hiddenTextStartColumn !== undefined ? { column: hiddenTextStartColumn, length: textBufferLine.length + 1 - hiddenTextStartColumn } : undefined);
 		this.additionalLinesWidget.updateLines(ghostText.lineNumber, additionalLines, ghostText.additionalReservedLineCount);
 
+		this.audioCueService.playAudioCue(AudioCue.inlineSuggestion).then(() => {
+			if (this.editor.getOption(EditorOption.screenReaderAnnounceInlineSuggestion)) {
+				const lineText = this.editor.getModel()?.getLineContent(ghostText.lineNumber);
+				if (lineText) {
+					alert(ghostText.renderForScreenReader(lineText));
+				}
+			}
+		});
 		if (0 < 0) {
```

Comment:
Note that you are making the assumption that the `update` method is only called on proper changes. However, that is not the semantics of the `update` method, it just makes sure the state is up to date.

Reply:
Ok is there some other place you'd recommend I put this? I find this code a little confusing to understand

Reply:
figured it out